README.md

```diff
@@ -1,2 +1,29 @@
 # migration
 Migration tools for TiKV, e.g. online bulk load.
+
+# How to build from source
+
+```
+mvn clean package
+```
+# Quick Start
```

Comment:
```suggestion

# Quick Start
```

I think you can elaborate a bit here for a complete example, from preparing the environment and step by step and what output we want to see.

Reply:
Updated in another comment.

Reply:
Fixed in commit [[BulkLoad: fix README.md and keys must be added in order exception](https://github.com/tikv/migration/pull/1/commits/3c0520a1d4aefee81148474a77b47ff0fcacc6da)]